lib/urbanopt/scenario/default_reports/reporting_period.rb

```diff
@@ -35,15 +35,197 @@ module Scenario
+          result[:demand_cost] = @demand_cost if @demand_cost
+          result[:comfort_result] = @comfort_result if @comfort_result 
+          result[:time_setpoint_not_met_during_occupied_cooling] = @time_setpoint_not_met_during_occupied_cooling if @time_setpoint_not_met_during_occupied_cooling 
+          result[:time_setpoint_not_met_during_occupied_heating] = @time_setpoint_not_met_during_occupied_heating if @time_setpoint_not_met_during_occupied_heating 
+          result[:time_setpoint_not_met_during_occupied_hours] = @time_setpoint_not_met_during_occupied_hours if @time_setpoint_not_met_during_occupied_hours
+
           return result
         end
-       
+        
+        def self.merge_reporting_period(existing_period, new_period)
+                            
+          # modify the existing_period by summing up the results
+          existing_period.total_site_energy += new_period.total_site_energy
+          existing_period.total_source_energy += new_period.total_source_energy
+          existing_period.net_site_energy += new_period.net_site_energy
+          existing_period.net_source_energy += new_period.net_source_energy
+          existing_period.net_utility_cost += new_period.net_utility_cost
+          existing_period.electricity += new_period.electricity
+          existing_period.natural_gas += new_period.natural_gas
+          existing_period.additional_fuel += new_period.additional_fuel
+          existing_period.district_cooling += new_period.district_cooling
+          existing_period.district_heating += new_period.district_heating
+          existing_period.water += new_period.water
+          existing_period.electricity_produced += new_period.electricity_produced
+          existing_period.heating += new_period.heating
+          existing_period.cooling += new_period.cooling
+          existing_period.interior_lighting += new_period.interior_lighting
+          existing_period.exterior_lighting += new_period.exterior_lighting
+          existing_period.interior_equipment += new_period.interior_equipment
+          existing_period.exterior_equipment += new_period.exterior_equipment
+          existing_period.fans += new_period.fans
+          existing_period.pumps += new_period.pumps
+          existing_period.heat_rejection += new_period.heat_rejection
+          existing_period.humidification += new_period.humidification
+          existing_period.heat_recovery += new_period.heat_recovery
+          existing_period.water_systems += new_period.water_systems
+          existing_period.refrigeration += new_period.refrigeration
+          existing_period.generators += new_period.generators
+          existing_period.photovoltaic += new_period.photovoltaic
+          existing_period.usage_cost += new_period.usage_cost
+          existing_period.demand_cost += new_period.demand_cost
+          existing_period.time_setpoint_not_met_during_occupied_cooling += new_period.time_setpoint_not_met_during_occupied_cooling
```

Comment:
time_setpoint_not_met seems a bit odd to add up, but I guess it is ok.  Thoughts @rawadelkontar ?

Reply:
I agree, it doesn't make sense to report the unmet hours.  Different buildings have different conditioning systems and  system sizes, so adding the unmet hours doesn't tell alot.